test/sde/sde_convergence_dynamical.jl

```diff
@@ -0,0 +1,31 @@
+using StochasticDiffEq, DiffEqNoiseProcess, Test, DiffEqDevTools, Random
```

Comment:
Can you move this file to the weak tests folder `weak_convergence`?

And also move the tests to buildkite by adding them as an additional group to:
```julia
if !is_APPVEYOR && GROUP == "WeakConvergence7"
    @time @safetestset "Dynamical SDE Tests" begin include("weak_convergence/sde_convergence_dynamical.jl") end
  end
```
https://github.com/SciML/StochasticDiffEq.jl/blob/master/test/runtests.jl

and also here: https://github.com/SciML/StochasticDiffEq.jl/blob/master/.buildkite/pipeline.yml